moped-editor/src/utils/mapHelpers.js

```diff
@@ -161,12 +185,38 @@ export const createZoomBbox = featureCollection => {
 };
 
 /**
- * Get the IDs from the layerConfigs object to set as interactive in the map components
+ * Get the layer names from the layerConfigs object for which isClickEditable is true
+ * @return {Array} List of source layer names that have features that can be added or removed by clicking
+ */
+export const getClickEditableLayerNames = () =>
+  Object.entries(mapConfig.layerConfigs).reduce(
+    (acc, [sourceLayerName, config]) =>
+      config.isClickEditable ? [...acc, sourceLayerName] : acc,
+    []
+  );
+
+/**
+ * Get the IDs from the layerConfigs object to set as interactive in the edit map
+ * Edit map needs all layers to be interactive to let users select features
  * @return {Array} List of layer IDs to be set as interactive (hover, click) in map
  */
-export const getInteractiveIds = () =>
+export const getEditMapInteractiveIds = () =>
   Object.values(mapConfig.layerConfigs).map(config => config.layerIdName);
 
+/**
+ * Get the IDs from the layerConfigs object to set as interactive in the summary map
+ * Summary map only needs layers in the project extent to be interactive
+ * @return {Array} List of layer IDs to be set as interactive (hover, click) in map
+ */
+export const getSummaryMapInteractiveIds = featureCollection => [
+  ...new Set(
+    featureCollection.features.map(
+      feature =>
+        mapConfig.layerConfigs[feature.properties.sourceLayer].layerIdName
+    )
+  ),
+];
+
```

Comment:
Brilliant use of sets!  🔢  ⚡  👍

Reply:
Totally stole this from one of your PRs 😄